topics/imaging/tutorials/process-image-bioimageio/tutorial.md

```diff
@@ -0,0 +1,182 @@
+>
+{: .tip}
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+>
+> 2. Download the following image and import it into your Galaxy history.  
+>    For the purpose of this tutorial, we will use one image to test only one of the 11 available models:
+>
+>    - [`test_image_nuclei.tif`](../../images/process-image-bioimageio/input_nucleisegboundarymodel.png)
+>
+>    If you are importing the image via URL:
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    If you are importing the image from the shared data library:
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets appropriately if needed (e.g. `"BioImage.IO model"`, `"Test image"`)
+>
+> 4. Confirm the datatypes are correct (`pt` for the model, `tiff` or `png` for the image)
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Import the BioImage.IO model from the Galaxy file repository:
+>
+>    - Click on **Upload Data**
+>    - Go to the **Choose remote files** tab
+>    - Navigate to: `ML models` → `bioimaging-models`
+>    - Select the desired model file (for this tutorial, choose `nucleisegmentationboundarymodel.pt`)
+>    - Click **Import** to add it to your history
+>
+>    If you are importing the model from the shared data library:
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+{: .hands_on}
+
+
+## Run the model on your image
+
+> <hands-on-title> Run BioImage.IO model </hands-on-title>
+>
+> 1. {% tool [Process image using a BioImage.IO model](toolshed.g2.bx.psu.edu/repos/bgruening/bioimage_inference/bioimage_inference/2.4.1+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"BioImage.IO model"*: `nucleisegmentationboundarymodel.pt`
+>    - {% icon param-file %} *"Input image"*: `input_nucleisegboundarymodel.png`
+>    - {% icon param-text %} *"Size of the input image"*: `256,256,1,1`
+>    - {% icon param-select %} *"Axes of the input image"*: `bcyx`
+>
+>    > <comment-title>Axes and size</comment-title>
+>    >
+>    > You must enter the input size and axes that match the model. This information is provided in the model's RDF file on BioImage.IO.
+>    {: .comment}
+>
+{: .hands_on}
+
+The model will process the input image and generate two outputs:
+- A predicted image (`.tif`)
+- A predicted tensor matrix (`.npy`)
+
+Below is an example output from the NucleiSegmentationBoundaryModel.
+
+![Example output from nuclei segmentation model](../../images/process-image-bioimageio/output-nucleus-seg-model.png "Predicted output – Nucleus Segmentation")
```

Comment:
Is this the TIF or the NPY, or both/neither? Please clarify.

Also, some interpretation of the result should be added here, since this was promised in the introduction :)
> You will learn how to upload and configure the model, set the correct input parameters, **and interpret the output files.**